### PR TITLE
[RFR] Add /setup route and view

### PR DESCRIPTION
**Pull request by eschaefer**
- [x] Set redirect after Slack auth to `/setup`

### CODE REVIEW

**Review by nburkley — APPROVED** [review]
👍

File: web/templates/setup/index.html.eex [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          <% else %>
+            <span class="steps__number">2</span>
+          <% end %>
+        </div>
+        <div class="steps__right">
+          <h3 class="steps__title"><strong>Step 2 &ndash;</strong> Authenticate Google Analytics</h3>
+          <%= if @current_team && @current_team.google_token do %>
+            <span>Already Linked</span>
+          <% else %>
+            <a href="#">Link your Google account.</a>
```
Comment (eschaefer) [inline web/templates/setup/index.html.eex]: Need a real button here.

File: web/templates/setup/index.html.eex [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  <% end %>
+</div>
+
+<div class="bottom">
+  <div class="bottom__offset container">
+    <h2 class="bottom__title">Welcome to POST Bot:</h2>
+    <ul class="steps steps--setup">
+      <li class="steps__item">
+        <div class="steps__left">
+          <%= if @current_team && @current_team.google_token do %>
```
Comment (eschaefer) [inline web/templates/setup/index.html.eex]: Need a different check here. How to determine if only Slack is authenticated?

Comment (nburkley) [reply web/templates/setup/index.html.eex]: If the user has been authenticated via slack, `@current_team` will be present, so just `if @current_team` should do it

File: config/dev.secret.exs.sample [hunk 3]
```diff
@@ -1,12 +0,0 @@
...
-config :espi_dni, EspiDni.Plugs.RequireSlackToken,
-  slack_token: "getfromslack"
-
-config :ueberauth, Ueberauth.Strategy.Slack.OAuth,
-  client_id: "slack-client-id",
-  client_secret: "slack-client-secret"
-
-config :ueberauth, Ueberauth.Strategy.Google.OAuth,
-  client_id: "google-client-id",
-  client_secret: "google-client-secret"
```
Comment (nburkley) [inline config/dev.secret.exs.sample]: this shouldn't be deleted, I'm guessing it was committed by mistake

**Review by nburkley — APPROVED** [review]

File: config/dev.secret.exs.sample (hunk 3 above)
Comment (eschaefer) [reply config/dev.secret.exs.sample]: Yeah, sorry - should have been stashed.

**Pull request merged**